src/configuration/iconfiguration.ts

```diff
@@ -205,4 +205,16 @@ export interface IConfiguration {
   insertModeKeyBindingsNonRecursive: IKeyRemapping[];
   otherModesKeyBindings: IKeyRemapping[];
   otherModesKeyBindingsNonRecursive: IKeyRemapping[];
+
+  /**
+   * Cursor movement options
+   */
+  stayOnYank: boolean;
```

Comment:
Please also add these configurations to https://github.com/VSCodeVim/Vim/blob/c8d6d34a381949c5bbaf4ce7330fcc1497a12560/test/testConfiguration.ts